src/components/FormatSettings/components/Formatters/NumericFormatter/NumericFormatter.tsx

```diff
@@ -94,7 +109,17 @@ const FormatValues: FC<Props> = ({ formatValue, onUpdateFormatValue }) => {
         label: t(DEFAULT_FORMATTER_PATTERN.label),
         value: DEFAULT_FORMATTER_PATTERN.value,
       },
-      ...patterns,
+      ...patterns.map((pattern) => {
+        if (pattern.value === FULL_NUMBER_PATTERN) {
```

Comment:
not necessary as `full number` should be removed. valid version of this part is on `develop` branch